src/printer.js

```diff
@@ -1776,6 +1789,31 @@ function genericPrintNoParens(path, options, print) {
     // supported by the pretty-printer.
     case "DeclaredPredicate":
       return concat(["%checks(", path.call(print, "value"), ")"]);
+    case "TSAnyKeyword":
+      return "any";
+    case "TSNumberKeyword":
+      return "number";
+    case "TSObjectKeyword":
+      return "object";
+    case "TSStringKeyword":
+      return "string";
+    case "TSVoidKeyword":
+      return "void";
+    case "TSFunctionType":
+      console.log('TSFunctionType', path.getValue())
+      return options.originalText.slice(util.locStart(path.getValue()), util.locEnd(path.getValue()));
+    case "TSArrayType":
+      return concat([path.call(print, "elementType"), "[]"]);
+    case "TSTypeLiteral": 
+      console.log('TSTypeLiteral', path.getValue())
+      return options.originalText.slice(util.locStart(path.getValue()), util.locEnd(path.getValue()));
+    case "TSPropertySignature":
+      console.log('TSPropertySignature', path.getValue())
+      return "TSPropertySignature"
+    case "TSParameterProperty":
+    case "TSTypeReference":
+      console.log('TSParameterProperty', path.getValue())
+      return options.originalText.slice(util.locStart(path.getValue()), util.locEnd(path.getValue()));
```

Comment:
I don't know actually there are upsides and downsides with both. The way I see it:

Adding that would allows us to pull more tests as we can pull those that covers what we want but still has missing features. That gives us better test coverage on the features implemented so far. However, one nice thing with only pulling completely supported test files is that it's very easy to diff for files that are not pulled yet. That might mean though that we have to come back to features that have been added if they break in some edge case that isn't tested yet.
Either way is fine for me.